src/components/projects.js

```diff
@@ -13,7 +13,10 @@ const Projects = ({ projectImgs }) => {
       <h2 className="text-center">PROJECTS</h2>
       <div className="section-content">
         <div className="subheader">
-          <FormGroup controlId="formControlsSelect">
+          <FormGroup controlId="project-type-select">
```

Comment:
Fix: Form is not accessible because we are unable to tab into the form and select an item from the drop down menu